### PR TITLE
162 2118749 proxy disconnected

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**
https://bugzilla.redhat.com/show_bug.cgi?id=2118749

### CODE REVIEW

File: doc-Service-Telemetry-Framework/assemblies/assembly_preparing-your-ocp-environment-for-stf.adoc [hunk 1]
```diff
@@ -6,19 +6,20 @@ ifdef::context[:parent-context: {context}]
...
 * Ensure that enough resources are available to run the Operators and the application containers. For more information, see <<resource-allocation_assembly-preparing-your-ocp-environment-for-stf>>.
-
+* Ensure that you have a fully connected network environment. For more information, see xref:con-network-considerations-for-service-telemetry-framework_assembly-preparing-your-ocp-environment-for-stf[].
 ifeval::["{build}" == "upstream"]
-* {ProjectShort} uses ElasticSearch to store events, which requires a larger than normal `vm.max_map_count`. The `vm.max_map_count` value is set by default in {OpenShift}. For more information about how to edit the value of `vm.max_map_count`, see <<node-tuning-operator_assembly-preparing-your-ocp-environment-for-stf>>.
+* {ProjectShort} uses ElasticSearch to store events, which requires a larger than normal `vm.max_map_count` value. The `vm.max_map_count` value is set by default in {OpenShift}. For more information about how to edit the value of `vm.max_map_count`, see <<node-tuning-operator_assembly-preparing-your-ocp-environment-for-stf>>.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/assemblies/assembly_preparing-your-ocp-environment-for-stf.adoc:15]:
```suggestion
* {ProjectShort} uses Elasticsearch to store events, which requires a larger than normal `vm.max_map_count` value. The `vm.max_map_count` value is set by default in {OpenShift}. For more information about how to edit the value of `vm.max_map_count`, see <<node-tuning-operator_assembly-preparing-your-ocp-environment-for-stf>>.
```

**Review by leifmadsen — APPROVED** [review]

**Comment by JoanneOFlynn2018** [comment]
Awaiting peer review

File: doc-Service-Telemetry-Framework/assemblies/assembly_preparing-your-ocp-environment-for-stf.adoc [hunk 2]
```diff
@@ -6,19 +6,20 @@ ifdef::context[:parent-context: {context}]
 :context: assembly-preparing-your-ocp-environment-for-stf
 
 [role="_abstract"]
-To prepare your {OpenShift} environment for {Project} ({ProjectShort}), you must plan for persistent storage, adequate resources, and event storage:
+To prepare your {OpenShift} environment for {Project} ({ProjectShort}), you must plan for persistent storage, adequate resources, event storage, and network considerations:
 
 * Ensure that persistent storage is available in your {OpenShift} cluster for a production grade deployment. For more information, see <<persistent-volumes_assembly-preparing-your-ocp-environment-for-stf>>.
```
Comment (mickogeary) [inline doc-Service-Telemetry-Framework/assemblies/assembly_preparing-your-ocp-environment-for-stf.adoc:11]:
```suggestion
* Ensure that you have persistent storage available in your {OpenShift} cluster for a production grade deployment. For more information, see <<persistent-volumes_assembly-preparing-your-ocp-environment-for-stf>>.
```

File: doc-Service-Telemetry-Framework/modules/con_support-for-stf.adoc [hunk 3]
```diff
@@ -30,4 +6,4 @@ Red Hat supports the two most recent versions of {Project} ({ProjectShort}). Ear
 
 Red Hat supports the core Operators and workloads, including {MessageBus}, AMQ Certificate Manager, Service Telemetry Operator, and Smart Gateway Operator. Red Hat does not support the community Operators or workload components, inclusive of ElasticSearch, Prometheus, Alertmanager, Grafana, and their Operators.
 
-{ProjectShort} does not work in {OpenShift} disconnected environments due to dependencies on components that are not yet available for installation in a disconnected environment.
+{ProjectShort} works only in a fully connected network environment. {ProjectShort} does not work in Red Hat OpenShift Container Platform-disconnected environments or network proxy environments.
```
Comment (mickogeary) [inline doc-Service-Telemetry-Framework/modules/con_support-for-stf.adoc:9]:
```suggestion
You can only deploy {ProjectShort} a fully connected network environment. You cannot deploy {ProjectShort} in Red Hat OpenShift Container Platform disconnected environments or network proxy environments.
```

File: doc-Service-Telemetry-Framework/modules/con_support-for-stf.adoc [hunk 4]
```diff
@@ -30,4 +6,4 @@ Red Hat supports the two most recent versions of {Project} ({ProjectShort}). Ear
 
 Red Hat supports the core Operators and workloads, including {MessageBus}, AMQ Certificate Manager, Service Telemetry Operator, and Smart Gateway Operator. Red Hat does not support the community Operators or workload components, inclusive of ElasticSearch, Prometheus, Alertmanager, Grafana, and their Operators.
```
Comment (mickogeary) [inline doc-Service-Telemetry-Framework/modules/con_support-for-stf.adoc:7]:
```suggestion
Red Hat supports the core Operators and workloads, including {MessageBus}, AMQ Certificate Manager, Service Telemetry Operator, and Smart Gateway Operator. Red Hat does not support the community Operators or workload components, such as ElasticSearch, Prometheus, Alertmanager, Grafana, and their Operators.
```

File: doc-Service-Telemetry-Framework/modules/con_network-considerations-for-service-telemetry-framework.adoc [hunk 5]
```diff
@@ -0,0 +1,6 @@
+:_content-type: CONCEPT
+
+[id="con-network-considerations-for-service-telemetry-framework_{context}"]
+= Network considerations for Service Telemetry Framework
+
+{Project} ({ProjectShort}) only works in a fully connected network environment. {ProjectShort} does not work in {OpenShift} disconnected environments or network proxy environments.
```
Comment (mickogeary) [inline doc-Service-Telemetry-Framework/modules/con_network-considerations-for-service-telemetry-framework.adoc:6]:
```suggestion
You can only deploy {Project}({ProjectShort}) in a fully connected network environment. You cannot deploy {ProjectShort} in Red Hat OpenShift Container Platform disconnected environments or network proxy environments.
```

**Review by mickogeary — APPROVED** [review]

File: doc-Service-Telemetry-Framework/assemblies/assembly_installing-the-core-components-of-stf.adoc [hunk 6]
```diff
@@ -18,7 +18,7 @@ You can use Operators to load the {Project} ({ProjectShort}) components and obje
 
 * An {OpenShift} version inclusive of {SupportedOpenShiftVersion} through {NextSupportedOpenShiftVersion} is running.
 * You have prepared your {OpenShift} environment and ensured that there is persistent storage and enough resources to run the {ProjectShort} components on top of the {OpenShift} environment. For more information, see https://access.redhat.com/articles/4907241[Service Telemetry Framework Performance and Scaling].
-
+* Your environment is fully connected. {ProjectShort} does not work in {OpenShift} disconnected environments or network proxy environments.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/assemblies/assembly_installing-the-core-components-of-stf.adoc:21]:
```suggestion
* Your environment is fully connected. {ProjectShort} does not work in a {OpenShift}-disconnected environments or network proxy environments.
```

File: doc-Service-Telemetry-Framework/assemblies/assembly_preparing-your-ocp-environment-for-stf.adoc [hunk 7]
```diff
@@ -6,19 +6,20 @@ ifdef::context[:parent-context: {context}]
 :context: assembly-preparing-your-ocp-environment-for-stf
 
 [role="_abstract"]
-To prepare your {OpenShift} environment for {Project} ({ProjectShort}), you must plan for persistent storage, adequate resources, and event storage:
+To prepare your {OpenShift} environment for {Project} ({ProjectShort}), you must plan for persistent storage, adequate resources, event storage, and network considerations:
 
-* Ensure that persistent storage is available in your {OpenShift} cluster for a production grade deployment. For more information, see <<persistent-volumes_assembly-preparing-your-ocp-environment-for-stf>>.
+* Ensure that you have persistent storage available in your {OpenShift} cluster for a production grade deployment. For more information, see <<persistent-volumes_assembly-preparing-your-ocp-environment-for-stf>>.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/assemblies/assembly_preparing-your-ocp-environment-for-stf.adoc:11]:
```suggestion
* Ensure that you have persistent storage available in your {OpenShift} cluster for a production-grade deployment. For more information, see <<persistent-volumes_assembly-preparing-your-ocp-environment-for-stf>>.
```

**Pull request merged**